spec/requests/global_strong_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "spec_helper"
```

Comment:
I think this non-API controller test is not valuable, so can we drop this test and the related controller?

Reply:
Removed this spec.